platform/apple/bindings/GoldenGate/Utils/CommonPeer.swift

```diff
@@ -311,27 +284,4 @@ open class CommonPeer {
         )
         customBlasterConfiguration.accept(customConfiguration)
     }
-
-    deinit {
-        // unregister RemoteApi modules
-        remoteTestServer?.unregister(module: blasterService)
-
-        if let coapTestService = coapTestService {
-             remoteTestServer?.unregister(module: coapTestService)
-        }
-
-        if let coapClientService = coapClientService {
-            remoteTestServer?.unregister(module: coapClientService)
-        }
-    }
-}
-
-extension CommonPeer: StackConfigurable {
```

Comment:
Aren't these methods used outside RemoteApi?

Reply:
No, as `StackConfigurable` was used only in `StackRemoteAPI`.